cmd/client/main.go

```diff
@@ -107,6 +110,49 @@ var copyCmd = &cobra.Command{
 	},
 }
 
+var deleteCmd = &cobra.Command{
```

Comment:
Note refactoring done in edeb821
use this wrapper https://github.com/yezzey-gp/yproxy/blob/master/cmd/client/main.go#L27